validator_client/Cargo.toml

```diff
@@ -61,3 +66,5 @@ logging = { workspace = true }
 strum = { workspace = true }
 itertools = { workspace = true }
 fdlimit = "0.3.0"
+regex = "1.10.6"
```

Comment:
can remove this

Reply:
Removed in [ed63f70](https://github.com/sigp/lighthouse/pull/6568/commits/ed63f70deeffa2b553e1576208d00cbf1b9fd8b6)